internal/cli/atlas/privateendpoints/gcp/interfaces/create.go

```diff
@@ -0,0 +1,114 @@
+		var err error
+		opts.store, err = store.New(store.AuthenticatedPreset(config.Default()), store.WithContext(ctx))
+		return err
+	}
+}
+
+func (opts *CreateOpts) parseEndpoints() []*atlas.GCPEndpoint {
+	endpoints := make([]*atlas.GCPEndpoint, len(opts.endpoints))
+	for i, endpoint := range opts.endpoints {
+		s := strings.Split(endpoint, "@")
+		endpoints[i] = &atlas.GCPEndpoint{
+			EndpointName: s[0],
+			IPAddress:    s[1],
+		}
+	}
+	return endpoints
+}
+
+var createTemplate = "Interface endpoint '{{.EndpointGroupName}}' created.\n"
+
+func (opts *CreateOpts) Run() error {
+	r, err := opts.store.CreateInterfaceEndpoint(opts.ConfigProjectID(), provider, opts.privateEndpointServiceID, opts.newInterfaceEndpointConnection())
+	if err != nil {
+		return err
+	}
+
+	return opts.Print(r)
+}
+
+func (opts *CreateOpts) newInterfaceEndpointConnection() *atlas.InterfaceEndpointConnection {
+	return &atlas.InterfaceEndpointConnection{
+		EndpointGroupName: opts.privateEndpointGroupID,
+		GCPProjectID:      opts.gcpProjectID,
+		Endpoints:         opts.parseEndpoints(),
+	}
+}
+
+// mongocli atlas privateEndpoint(s) gcp interface(s) create <endpointGroupId> --endpointServiceId endpointServiceId --gcpProjectId gcpProjectId --endpoint endpointName1@ipAddress1,...,endpointNameN@ipAddressN [--projectId projectId].
+func CreateBuilder() *cobra.Command {
+	opts := &CreateOpts{}
+	cmd := &cobra.Command{
+		Use:     "create <endpointGroupId>",
+		Aliases: []string{"add"},
+		Short:   "Create a GCP private endpoint interface.",
+		Args:    require.ExactArgs(1),
+		Annotations: map[string]string{
+			"args":                "endpointGroupId",
+			"requiredArgs":        "endpointGroupId",
+			"endpointGroupIdDesc": "Unique identifier for the endpoint group.",
+		},
+		Example: `$ mongocli atlas privateEndpoints gcp interfaces create endpoint-1 --endpointServiceId 61eaca605af86411903de1dd --gcpProjectId mcli-private-endpoints --endpoint endpoint-0@10.142.0.2,endpoint-1@10.142.0.3,endpoint-2@10.142.0.4,endpoint-3@10.142.0.5,endpoint-4@10.142.0.6,endpoint-5@10.142.0.7`,
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return opts.PreRunE(
+				opts.ValidateProjectID,
+				opts.initStore(cmd.Context()),
+				opts.InitOutput(cmd.OutOrStdout(), createTemplate),
+			)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			opts.privateEndpointGroupID = args[0]
+			return opts.Run()
+		},
+	}
+	cmd.Flags().StringVar(&opts.privateEndpointServiceID, flag.EndpointServiceID, "", usage.EndpointServiceID)
+	cmd.Flags().StringVar(&opts.gcpProjectID, flag.GCPProjectID, "", usage.GCPProjectID)
+	cmd.Flags().StringSliceVar(&opts.endpoints, flag.Endpoint, []string{}, usage.Endpoint)
+	cmd.Flags().StringVar(&opts.ProjectID, flag.ProjectID, "", usage.ProjectID)
+	cmd.Flags().StringVarP(&opts.Output, flag.Output, flag.OutputShort, "", usage.FormatOut)
+	_ = cmd.MarkFlagRequired(flag.EndpointServiceID)
+	_ = cmd.MarkFlagRequired(flag.GCPProjectID)
```

Comment:
Question for team mates @andreaangiolillo @gssbzn : Surely we don't need to assign to the underscore?

ie. we could replace:
`	_ = cmd.MarkFlagRequired(flag.GCPProjectID)
`with simply:
`	cmd.MarkFlagRequired(flag.GCPProjectID)
`
(the code I have here was a straight copy-paste form the AWS code)

Reply:
I think that's a go thing - unassigned return values are caught as an error?

Reply:
`MarkFlagRequired` returns an error if the flag doesn't exists in the set of flags of the command, we should either check it or explicitly ignore it

Reply:
It compiles fine when I remove the assignment, so the language allows leaving it out.

But I get what Gustavo says: by assigning the returned error to underscore we are being more explicit - we are saying that we are aware of a potential error and deliberately ignoring it. 

So,  I'll keep it as-is, thanks!